server/routes/routes.test.js

```diff
@@ -96,6 +96,25 @@ describe('dag endpoints', () => {
 
     done()
   })
+
+  test('GET contracts returns list of contracts', async done => {
+    const body = {
+      contractAddress: '0xfffd933a0bc612844eaf0c6fe3e5b8e9b6c1d19f'
+    }
+
+    await request(app)
+      .post('/api/v0/contracts')
+      .send(body)
+      .expect(201)
+
+    await request(app)
+      .get('/api/v0/contracts')
+      .send()
+      .expect(200)
+      .expect(res => expect(res.body.length).toBeGreaterThan(0))
```

Comment:
Might be good to test the content of the response object. That way if we change what the smart contract looks like in the DB in a breaking way, we'll know for sure from failing tests

Reply:
True!